grails-app/controllers/org/pih/warehouse/JsonController.groovy

```diff
@@ -866,7 +866,8 @@ class JsonController {
                         valueText     : localizedName,
                         desc          : product.description,
                         inventoryItems: inventoryItemList,
-                        handlingIcons : product?.getHandlingIcons()
+                        handlingIcons : product?.getHandlingIcons(),
+                        color         : product?.color
```

Comment:
This is a super expensive convenience method. Could you see if there's a way to make this more performant?